main/tests/server/src/com/google/refine/importing/ImportingUtilitiesTests.java

```diff
@@ -82,4 +95,75 @@ private ObjectNode getNestedOptions(ImportingJob job, TreeImportingParserBase pa
         JSONUtilities.safePut(options, "recordPath", path);
         return options;
     }
+
+    private static String RESPONSE_BODY = "{\"code\":401," + "\"message\":\"Unauthorised\"" + "}";
+    private final static String MESSAGE = String.format("HTTP error %d : %s | %s", 401,
+            "Client Error", RESPONSE_BODY);
```

Comment:
Since both of these variables are only used in the test below, I would just define them there. If they really need to be declared as static attributes, then they should go at the beginning of the test class. Also I would remove the `+` concatenation operators and just declare `RESPONSE_BODY` as a single string.